components/Section/FeaturedProducts.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <div>
+    <div
+      class="absolute -top-10 -z-10 hidden h-full w-3/4 border-[20px] border-gray-100 md:block"
+    />
+    <div class="w-1/3 text-gray-400 sm:px-2 md:px-8">
+      <h2
+        class="mt-4 ml-4 text-4xl uppercase"
+        data-preview-id="#st_featured_products_title"
+      >
+        {{ data.st_featured_products_title }}
+      </h2>
+      <h3
+        v-if="data.st_featured_products_header"
+        class="ml-4"
+        data-preview-id="#st_featured_products_header"
+      >
+        {{ data.st_featured_products_header }}
+      </h3>
+    </div>
+    <div
+      class="grid h-4/5 gap-2 sm:grid-cols-2 lg:grid-cols-3 lg:pl-8 2xl:grid-cols-4"
+    >
+      <div
+        v-for="(product, index) in data.st_featured_products_picture"
+        :key="product.id"
+      >
+        <ElementsFeaturedProductItem
+          :product="product"
+          :image-source="imgSrc[index]"
+        />
+      </div>
+    </div>
+  </div>
+</template>
+<script setup lang="ts">
+import { DataEntry, Image } from 'fsxa-api/dist/types'
+interface FeaturedProducts {
+  st_featured_products_header: null | string
+  st_featured_products_picture: DataEntry[]
+  st_featured_products_title?: null | string
+}
```

Comment:
why are not these properties in camelCase

Reply:
Because those are the names of the keys in the data from fsxa api. This naming is consistent throughout the project.